configure.sh

```diff
@@ -85,7 +85,7 @@ else
 fi
 
 if [ "${CIRCT_ENABLED}" == "yes" ] ; then
-	CPPFLAGS_CIRCT="-DCIRCT=1 -I${CIRCT_PATH}/include"
```

Comment:
If I am not mistaken, then the `-Wno-error=comment` should go into `CXXFLAGS_CIRCT` and not `CPPFLAGS_CIRCT`.

Reply:
Also, why was this not caught by the CI? Is it because we only compile HLS backend with clang?

Reply:
That's the case.

Reply:
Changed to CXXFLAGS_CIRCT